bson.cpp

```diff
@@ -150,6 +157,14 @@ void VariantToBsonConverter::convertElement(bson_t *bson, const char *key, Varia
 		case KindOfRef:
 			convertElement(bson, key, *v.getRefData());
 			break;
+#if HIPPO_HHVM_VERSION >= 31700
+		case KindOfPersistentVec:
+		case KindOfPersistentDict:
+		case KindOfPersistentKeyset:
+		case KindOfVec:
+		case KindOfDict:
+		case KindOfKeyset:
+#endif
```

Comment:
I don't think so, as then I won't catch whenever they add a new type. I am going to expect some user to use one of these new ones, and then I am going to need to hunt where these new types come from and handle them. This is not the first time they've added types randomly...